qcodes/instrument_drivers/signal_hound/USB_SA124B.py

```diff
@@ -5,18 +5,20 @@
 
 from qcodes import Instrument, validators as vals
 
+log = logging.getLogger(__name__)
+
 
 class SignalHound_USB_SA124B(Instrument):
-    '''
+    """
     This is a direct port of the signal hound QTLab driver by Ramiro
-    Edited by Adriaan Rol
+    Edited by Adriaan Rol and others (use git blame for details)
```

Comment:
what is this comment useful for? i'd remove it since "git blame" is supposedly a known feature of git :)

Reply:
Yeah, I was trying to humorously point out that information about who wrote what does not really belong in a driver... From your comment I guess that I failed :) You can remove the whole shebang if you'd like.

Reply:
😸 ok, i get it. well, I think that such a note is good fun for github comments, but should not be part of the code. So, I suggest to remove this line.